src/services/TestDataService.ts

```diff
@@ -144,6 +145,7 @@ export class TestDataService {
         const productResponse = await this.AdminApiClient.post('./product?_response=detail', {
             data: basicProduct,
         });
+        expect(productResponse.ok()).toBeTruthy();
```

Comment:
We should always check this, if we read the data from the response. Otherwise we get a low level runtime error, which is harder to debug.

Reply:
@Phil23 do you know why you didn't add there expect here?